.github/workflows/codespell.yml

```
@@ -0,0 +1,16 @@
+# Helps catch spelling errors
+name: Codespell
+on: [ pull_request ]
+
+jobs:
+  codespell:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

```suggestion
      - uses: actions/checkout@e2f20e631ae6d7dd3b768f56a5d2af784dd54791 # v2.5.0
```

please use git hash instead of git tags and add the version as a comment, do the same for the others thanks!